### PR TITLE
feat: update swagger docs

### PR DESCRIPTION
**Pull request by rileydcampbell**

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -91,14 +94,14 @@
...
       "get": {
-        "tags": ["Dutch Auction"],
+        "tags": ["Dutch Order"],
         "summary": "Get nonce",
         "description": "Get current nonce for dutch auction orders. Given an address this endpoint will return the next valid nonce to be used in order creation.",
         "parameters": [
           {
-            "$ref": "#/components/parameters/limitParam"
+            "$ref": "#/components/parameters/addressParam"
           }
```
Comment (zhongeric) [inline swagger.json:105]: note that nonce also takes an optional chainId argument

File: swagger.json [hunk 2]
```diff
@@ -422,6 +430,7 @@
       },
       "PostOrderRequest": {
         "type": "object",
+        "required": ["encodedOrder", "signature", "chainId"],
```
Comment (zhongeric) [inline swagger.json:433]: should we add `quoteId` here too? they would get this from URA and it does help a lot in debugging

**Review by zhongeric — APPROVED** [review]

**Review by marktoda — APPROVED** [review]

File: swagger.json (hunk 2 above)
Comment (rileydcampbell) [reply swagger.json:433]: Hmmm we could but it's not actually required

File: swagger.json (hunk 1 above)
Comment (rileydcampbell) [reply swagger.json:105]: Added chainId here and forgot to also include on the GET /orders parameters

**Pull request merged**